src/engine/vcpkg.json

```diff
@@ -126,6 +127,10 @@
     {
       "name": "libarchive",
       "version": "3.7.7#0"
+    },
+    {
+      "name": "openssl",
+      "version": "3.1.0#0"
```

Comment:
3.1.0 not is the latest version, we can use the 3.4.0#0 if we dont have some limitation to change the minor version, or we can use the 3.1.4#1 if we need maintain the minor version.

Reply:
I selected the version closest to the one used in 4.x because of the OpenSSL helpers that I moved.

I'll try with 3.4.0#0